--- a/pcmanfm/main-win.ui
+++ b/pcmanfm/main-win.ui
@@ -268,13 +268,5 @@
-   <addaction name="actionNewTab"/>
+   <addaction name="actionMenu"/>
    <addaction name="actionGoBack"/>
    <addaction name="actionGoForward"/>
    <addaction name="actionGoUp"/>
-   <addaction name="actionReload"/>
-   <addaction name="separator"/>
-   <addaction name="actionIconView"/>
-   <addaction name="actionThumbnailView"/>
-   <addaction name="actionCompactView"/>
-   <addaction name="actionDetailedList"/>
-   <addaction name="separator"/>
    <addaction name="actionGo"/>
-   <addaction name="actionMenu"/>
